CDTDatastoreTests/CDTSessionCookieInterceptorTests.m

```diff
@@ -87,6 +87,16 @@ - (void)setUp
           }
 
         }];
+    
+    [OHHTTPStubs stubRequestsPassingTest:^BOOL(NSURLRequest *__nonnull request) {
```

Comment:
I think it is confusing having this stub in the setup when all the other stubs are done in the test case itself for the 403 cases. This stub is needed for your current test, but it is returning a 403 when I think you really want a different response (probably 200 for expired path, and 403 for non-expired path) - I think it would be better to keep the stubs together rather than splitting them like this, which is already causing some unexpected results (see later comments).

Reply:
Thanks for catching this - it was a 'left-over' stub from a removed test case.